### PR TITLE
fix(#1933): new help files structure

### PR DESCRIPTION
**Pull request by vomikan**
This change is [Reviewable](https://reviewable.io/reviews/moneymanagerex/moneymanagerex/1995)

### CODE REVIEW

File: docs/en/budget.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    <link href="../manual.css" rel="stylesheet" type="text/css">
+    <script src="../js/get_param.js"></script>
+    <script src="../js/toc.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
+    <script src="../js/back_to_top.js"></script>
+</head>
+<body onload="generateTOC(document.getElementById('toc'));">
+    <header>
+        <nav><small><a id="go_home" href="./index.html">Go to main help page</a></small></nav>
+        <a href="#" id="back-to-top"><img src="../en/backtotop.png" alt="Back to Top" /></a>
```
Comment (slodki) [inline docs/en/budget.html]:
```suggestion
        <a href="#" id="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
```

File: docs/en/grm.html [hunk 2]
```diff
@@ -3,22 +3,17 @@
...
+    <link href="../manual.css" rel="stylesheet" type="text/css">
+    <script src="../js/get_param.js"></script>
+    <script src="../js/toc.js"></script>
     <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
-    <script src="js/back_to_top.js"></script>
+    <script src="../js/back_to_top.js"></script>
 </head>
 <body onload="generateTOC(document.getElementById('toc'));">
     <header>
         <nav><small><a id="go_home" href="index.html">Go to main help page</a></small></nav>
```
Comment (slodki) [inline docs/en/grm.html]:
```suggestion
        <nav><small><a href="../index.html?lang=en">Go to main help page</a></small></nav>
```

File: docs/en/grm.html [hunk 3]
```diff
@@ -3,22 +3,17 @@
...
-        <script>
-            var lang = getParameterValue("lang");
-            if (lang)
-                document.getElementById('go_home').href = "index.html?lang=" + lang;
-        </script>
 
-        <a href="#" id="back-to-top"><img src="img/backtotop.png" alt="Back to Top" /></a>
+        <a href="#" id="back-to-top"><img src="../en/backtotop.png" alt="Back to Top" /></a>
```
Comment (slodki) [inline docs/en/grm.html]:
```suggestion
        <a href="#" id="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
```

File: docs/en/index.html [hunk 4]
```diff
@@ -0,0 +1,1646 @@
...
+            <a href="../index.html?lang=de"><span class="flag-icon flag-icon-de"></span> German</a>
+            <a href="../index.html?lang=fr"><span class="flag-icon flag-icon-fr"></span> French</a>
+            <a href="../index.html?lang=hu"><span class="flag-icon flag-icon-hu"></span> Hungarian</a>
+            <a href="../index.html?lang=it"><span class="flag-icon flag-icon-it"></span> Italian</a>
+            <a href="../index.html?lang=pl"><span class="flag-icon flag-icon-pl"></span> Polish</a>
+            <a href="../index.html?lang=ru"><span class="flag-icon flag-icon-ru"></span> Russian</a>
+            <a href="../index.html?lang=es"><span class="flag-icon flag-icon-es"></span> Spanish</a>
+        </small></nav>
+
+        <a href="#" id="back-to-top"><img src="../en/backtotop.png" alt="Back to Top" /></a>
```
Comment (slodki) [inline docs/en/index.html]:
```suggestion
        <a href="#" id="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
```

File: docs/en/index.html [hunk 5]
```diff
@@ -0,0 +1,1646 @@
...
+        (http://sqlitebrowser.sourceforge.net/) are two such
+        utilities. Once you open the database using these tools,
+        you can do anything you want with the data.
+    </p>
+
+    <h3>Is my data safe?</h3>
+
+    <p>Yes. Your data is completely safe.</p>
+    <p>
+        The data is self contained on your PC, (or USB
```
Comment (slodki) [inline docs/en/index.html]: equipo?

File: docs/en/stocks_and_shares.html [hunk 6]
```diff
@@ -3,22 +3,22 @@
...
         <nav><small><a id="go_home" href="index.html">Go to main help page</a></small></nav>
         <script>
             var lang = getParameterValue("lang");
             if (lang)
-                document.getElementById('go_home').href = "index.html?lang=" + lang;
+                document.getElementById('go_home').href = "../index.html?lang=" + lang;
         </script>
 
-        <a href="#" id="back-to-top"><img src="img/backtotop.png" alt="Back to Top" /></a>
+        <a href="#" id="back-to-top"><img src="../en/backtotop.png" alt="Back to Top" /></a>
```
Comment (slodki) [inline docs/en/stocks_and_shares.html]:
```suggestion
        <a href="#" id="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
```

File: docs/es/budget.html [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+    <link href="../manual.css" rel="stylesheet" type="text/css">
+    <script src="../js/get_param.js"></script>
+    <script src="../js/toc.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
+    <script src="../js/back_to_top.js"></script>
+</head>
+<body onload="generateTOC(document.getElementById('toc'));">
+    <header>
+        <nav><small><a id="go_home" href="index.html">Go to main help page</a></small></nav>
+        <a href="#" id="back-to-top"><img src="../en/backtotop.png" alt="Back to Top" /></a>
```
Comment (slodki) [inline docs/es/budget.html]:
```suggestion
        <a href="#" id="back-to-top"><img src="../img/backtotop.png" alt="Back to Top" /></a>
```

File: docs/es/budget.html [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+    <title>Budgeting</title>
+    <link href="../manual.css" rel="stylesheet" type="text/css">
+    <script src="../js/get_param.js"></script>
+    <script src="../js/toc.js"></script>
+    <script src='http://ajax.googleapis.com/ajax/libs/jquery/2.0.0/jquery.min.js'></script>
+    <script src="../js/back_to_top.js"></script>
+</head>
+<body onload="generateTOC(document.getElementById('toc'));">
+    <header>
+        <nav><small><a id="go_home" href="index.html">Go to main help page</a></small></nav>
```
Comment (slodki) [inline docs/es/budget.html]:
```suggestion
        <nav><small><a id="go_home" href="../index.html?lang=es">Go to main help page</a></small></nav>
```

File: docs/pl/grm.html [hunk 9]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```
Comment (slodki) [inline docs/pl/grm.html]: this file should be replaced by docs/grm.html?lang=pl

File: docs/en/index.html (hunk 5 above)
Comment (slodki) [inline docs/en/index.html]:
```suggestion
        var LANG = document.getElementsByTagName('html')[0].lang;
```

Comment (slodki) [inline docs/en/index.html]:
```suggestion
            var tr = document.getElementById("translator");
            tr.parentNode.removeChild(tr);
```

File: docs/pl/grm.html (hunk 9 above)
Comment (vomikan) [reply docs/pl/grm.html]: This file will replased this one.
https://crowdin.com/project/moneymanagerex/pl#
Now is very important to move English files into 'en' folder, then upload them into Crowbin.

File: docs/de/index.html [hunk 10]
```diff
@@ -2203,12 +2198,7 @@ <h2 class="i18n">Aktiva</h2>
     </p>
     <hr>
     <h2 class="i18n">Budgetierung</h2>
-    <a class="i18n" id="budget_href" href="http://desktop.moneymanagerex.org/budget.html">Budgetierung in MMEX</a>
-    <script>
-        var lang = getParameterValue("lang");
-        if (lang !== '')
-            document.getElementById('budget_href').href = "budget.html?lang=" + lang;
-    </script>
+    <a class="i18n" id="budget_href" href="../en/budget.html">Budgetierung in MMEX</a>
```
[the remark this answers is not in the document]
Comment (vomikan) [reply docs/de/index.html]: useless

File: docs/es/budget.html (hunk 8 above)
Comment (vomikan) [reply docs/es/budget.html]: Disagree. it should not be a language code.

File: docs/de/grm.html [hunk 11]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
```
[the remark this answers is not in the document]
Comment (vomikan) [reply docs/de/grm.html]: This files calls directly from mmex. If missing it will be page not found error.

File: docs/pl/grm.html (hunk 9 above)
Comment (vomikan) [reply docs/pl/grm.html]: It is easier to replace the file than to alter the script. Fake file is workaround (temporary solution).

**Pull request merged**